torchvision/models/optical_flow/_utils.py

```diff
@@ -19,8 +19,8 @@ def grid_sample(img: Tensor, absolute_grid: Tensor, mode: str = "bilinear", alig
     return F.grid_sample(img, normalized_grid, mode=mode, align_corners=align_corners)
 
 
-def make_coords_grid(batch_size: int, h: int, w: int):
-    coords = torch.meshgrid(torch.arange(h), torch.arange(w), indexing="ij")
+def make_coords_grid(batch_size: int, h: int, w: int, device: torch.device = torch.device("cpu")):
```

Comment:
@TeodorPoncu I notice on another PR @jdsgomes comment this: https://github.com/pytorch/vision/pull/6554#discussion_r969626378

We might want to have similar thing here.